alonzo/impl/src/Cardano/Ledger/Alonzo/Scripts.hs

```diff
@@ -208,10 +208,14 @@ instance NFData Prices
 
 -- | Compute the cost of a script based upon prices and the number of execution
 -- units.
-scriptfee :: Prices -> ExUnits -> Coin
+scriptfee :: Prices -> ExUnits -> SubCoin
 scriptfee (Prices pr_mem pr_steps) (ExUnits mem steps) =
   (mem <×> pr_mem) <+> (steps <×> pr_steps)
 
+-- | Same as `scriptfee`, but returns the fee rounded to the Coin precision.
+txscriptfee :: Prices -> ExUnits -> Coin
+txscriptfee ps = coin . scriptfee ps
```

Comment:
let's coordinate with @polinavino (who is making the corresponding spec change) to keep the names aligned.

Reply:
Cool I got the `txscriptfee` name from the spec, I can certainly change it to whatever it needs to be

Reply:
oops! you had it correct, thank you!